App/AntSimulator.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System.Collections.Generic;
+using AntEngine;
+using AntEngine.Entities.Ants;
+using AntEngine.Entities.Colonies;
+using AntEngine.Utils.Maths;
+using App.Renderers;
+using App.Renderers.EntityRenderers;
+using App.UIElements;
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using Microsoft.Xna.Framework.Input;
+using Vector2 = System.Numerics.Vector2;
+
+
+namespace App
+{
+    public class AntSimulator : Game
+    {
+        private GraphicsDeviceManager _graphics;
+        private SpriteBatch _spriteBatch;
+
+        private World world;
+        private List<IRenderer> renderers;
+        
+        public AntSimulator()
+        {
+            _graphics = new GraphicsDeviceManager(this);
+            Content.RootDirectory = "Content";
+            IsMouseVisible = true;
+            world = new World(Vector2.One * 500);
+            renderers = new List<IRenderer>();
+        }
+
+        protected override void Initialize()
+        {
+            SimFrame mainSimFrame = new SimFrame();
+            
+            renderers.Add(mainSimFrame);
+            mainSimFrame.AddRenderer(new EntityRenderer(new Ant("EntityTest",
+                new Transform(new Vector2(50, 50), 0, new Vector2(30, 30)), world)));
+            mainSimFrame.AddRenderer(new AntRenderer(new Ant("AntTest",
+                new Transform(new Vector2(150, 150), 0, new Vector2(20, 20)), world)));
+            mainSimFrame.AddRenderer(new ColonyRenderer(new Colony("ColonyTest",
+                new Transform(new Vector2(300, 170), 0, new Vector2(64, 64)), world, (s, t , w, c) => new Ant(world))));
```

Comment:
Even if this is to test, maybe seperate the initialisation of the renderers and the entities. (not put a new in a new)

Reply:
If you want, this is just for testing so it will be removed when actual ant generation is added.